collectors/gateway/colconfig.yaml

```diff
@@ -54,6 +54,11 @@ exporters:
     tls:
       insecure: true
 
+  otlphttp/jaeger:
+    endpoint: http://jaeger-all-in-one-collector.jaeger.svc.cluster.local:4318
```

Comment:
```suggestion
    endpoint: http://jaeger-all-in-one.jaeger.svc.cluster.local:4318
```

Isn't it supposed to be the following based on the deployment file? I don't see where the name appends-collector

Reply:
![image](https://github.com/user-attachments/assets/85da59a9-fff2-436b-90a9-1c2a1a3bf519)